### PR TITLE
commands: add account related commands

### PR DESCRIPTION
**Pull request by sergiusens**
- [x] Have you followed the [guidelines for contributing](https://github.com/snapcore/snapcraft/blob/master/CONTRIBUTING.md)?
- [x] Have you signed the [CLA](http://www.ubuntu.com/legal/contributors/)?
- [x] Have you successfully run `./runtests.sh static`?
- [x] Have you successfully run `./runtests.sh tests/unit`?

-----
CRAFT-983

### CODE REVIEW

**Comment by sergiusens** [comment]
7b33d9b18422275d759218b1ac7a73ab3df13c01 has its own PR on #3698

**Comment by sergiusens** [comment]
de1280639559403e085da53ce5f8db74c158cf88 has its own PR on #3699

**Comment by sergiusens** [comment]
234d31e4c70001c88270032696332489f2abb95a has a PR #3701

File: snapcraft/commands/account.py [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+            # Now that the file has been written, we can just make it
+            # owner-readable
+            os.chmod(parsed_args.login_file, stat.S_IRUSR)
+
+            message = f"Exported login credentials to {parsed_args.login_file!r}"
+
+        emit.message(message)
+
+
+class StoreWhoCommand(BaseCommand):
```
Comment (cmatsuoka) [inline snapcraft/commands/account.py:225]: Shouldn't this be  `StoreWhoamiCommand` for consistency?

**Review by cmatsuoka — APPROVED** [review]
Couldn't see anything obviously wrong, but maybe a class name is inconsistent.

File: snapcraft/commands/account.py (hunk 1 above)
Comment (sergiusens) [reply snapcraft/commands/account.py:225]: Yeah, it should. I think I was sort of regretting not sticking with "who" as is the unix command

**Pull request merged**